xlsxconverter/XLSXConverter2.js

```diff
@@ -426,11 +426,16 @@ var XLSXConverter = {};
 			// odk time stamp -- UTC timestamp string with 00:00:00.000 time.
             "type": "string",
 			"elementType": "date"
+        },
+        "dateshort": {
```

Comment:
Very open to a better name than dateshort.

Reply:
why not be specific for the base type so it makes sense to someone looking at the code. 'date_no_time_zone"